src/Lib.P2P/Protocols/ProtocolRegistry.cs

```diff
@@ -0,0 +1,71 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System;
+using System.Collections.Generic;
+
+namespace Lib.P2P.Protocols
+{
+    /// <summary>
+    ///   Metadata on <see cref="IPeerProtocol"/>.
+    /// </summary>
+    public static class ProtocolRegistry
+    {
+        /// <summary>
+        ///   All the peer protocols.
+        /// </summary>
+        /// <remarks>
+        ///   The key is the name and version of the peer protocol, like "/multiselect/1.0.0".
+        ///   The value is a Func that returns an new instance of the peer protocol.
+        /// </remarks>
+        public static Dictionary<string, Func<IPeerProtocol>> Protocols;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)